docs/extending-typespec/basics.md

```diff
@@ -205,13 +205,36 @@ TypeSpec libraries can contain more than just types. Read the subsequent topics
 
 TypeSpec provides a testing framework to help testing libraries. Examples here are shown using `mocha` but any other JS test framework can be used.
 
+### Add devDependencies
+
+Verify that you have the following in your `package.json`:
+
+```
+  ...,
+  "devDependencies": {
+    "@types/node": "~18.11.9",
+    ...,
+    // if you want to include the mocha test framework
+    "@types/mocha": "~10.0.1",
+    "mocha": "~10.2.0",
+    "mocha-junit-reporter": "~2.2.1",
```

Comment:
I don't think I would recommend setting up those reporters, this is too much unecessary stuff, It is only useful if you want to publish test to something that consume junit(ado)